packages/breakdown-trees/src/breakdown-trees-react.ts

```diff
@@ -32,3 +32,4 @@ export * from "./Views/OptionItemHandlers";
 export * from "./Views/RelatedElementIdsProvider";
 export * from "./Views/NodeRenderers/FunctionalTreeNodeRenderer";
 export * from "./Views/TreeNodeFunctionsToolbar";
+export * from "./test/functionalityProviders/FunctionalityProviderTestUtils";
```

Comment:
does this really need to be exported?

Reply:
Yes, the code here are used from building-ui-components to test some functionalityProviders that we are passing to this package as additionaFunctionalityProvider. So instead of rewriting the same we can use it from there.